lib/yard-sorbet/sig_handler.rb

```diff
@@ -64,23 +70,23 @@ def process
     enhance_tag(docstring, :abstract, parsed_sig)
     enhance_tag(docstring, :return, parsed_sig)
     if method_node.type != :command
-      parsed_sig.params.each do |name, types|
-        enhance_param(docstring, name, types)
+      parsed_sig.params.each do |name, parsed_param|
+        enhance_param(docstring, name, parsed_param)
       end
     end
     method_node.docstring = docstring.to_raw
     YARDSorbet::Directives.add_directives(method_node.docstring, directives)
     sig_node.docstring = nil
   end
 
-  sig { params(docstring: YARD::Docstring, name: String, types: T::Array[String]).void }
-  private def enhance_param(docstring, name, types)
+  sig { params(docstring: YARD::Docstring, name: String, parsed_param: ParsedParam).void }
+  private def enhance_param(docstring, name, parsed_param)
     tag = docstring.tags.find { |t| t.tag_name == 'param' && t.name == name }
     if tag
       docstring.delete_tag_if { |t| t == tag }
-      tag.types = types
+      tag.types = parsed_param.types
     else
-      tag = YARD::Tags::Tag.new(:param, '', types, name)
+      tag = YARD::Tags::Tag.new(:param, parsed_param.docstring, parsed_param.types, name)
```

Comment:
Not sure if it matters, but here you change the default docstring from `''` to `nil` (unless `p.docstring` down below returns `''`)

Reply:
good call, will fix